gateway-service/src/test/java/org/zowe/apiml/gateway/security/login/zosmf/ZosmfAuthenticationProviderTest.java

```diff
@@ -138,11 +140,40 @@ void loginWithExistingUser() {
             new ZosmfAuthenticationProvider(authenticationService, zosmfService, authConfigurationProperties);
 
         mockZosmfRealmRestCallResponse();
+        Authentication tokenAuthentication
+            = zosmfAuthenticationProvider.authenticate(usernamePasswordAuthentication);
+
+        assertTrue(tokenAuthentication.isAuthenticated());
+        assertEquals(USERNAME, tokenAuthentication.getPrincipal());
+    }
+
+    @Test
+    void changePasswordAndAuthenticate() {
```

Comment:
Could we do something about the test? It doesn't conform to our new structure. 
Even potentially prepare separate PR to clean this class and then add this new test.